pubspec.yaml

```diff
@@ -10,7 +10,8 @@ environment:
 
 dependencies:
   js: '>=0.6.3 <0.7.0'
-  rxdart: '>=0.26.0 <0.28.0'
+  rxdart: '>=0.27.0 <0.28.0'
```

Comment:
supporting rxdart 0.26.0 is a must because it has to work nicely with flutter blue.

Reply:
Ok. let me know whether we could simple get rid of it or implement something that would make the linter happy for both versions

Reply:
I fixed using the compat trick. I tested on both rxdart 0.26 and rxdart 0.27.

Reply:
Thanks for that, I'll also look into just removing the rxdart dependency because less dependencies do cause less headache.